client/header/activity-panel/unread-indicators.js

```diff
@@ -31,21 +40,15 @@ export function getUnreadNotes( select ) {
 
 	// @todo This method would be more performant if we ask only for 1 item per page with status "unactioned".
 	// This change should be applied after having pagination implemented.
-	const notesQuery = {
-		page: 1,
-		per_page: QUERY_DEFAULTS.pageSize,
-		status: 'unactioned',
-		type: QUERY_DEFAULTS.noteTypes,
-		orderby: 'date',
-		order: 'desc',
-	};
 
 	// Disable eslint rule requiring `latestNotes` to be defined below because the next two statements
 	// depend on `getNotes` to have been called.
 	// eslint-disable-next-line @wordpress/no-unused-vars-before-return
-	const latestNotes = getNotes( notesQuery );
-	const isError = Boolean( getNotesError( 'getNotes', [ notesQuery ] ) );
-	const isRequesting = isResolving( 'getNotes', [ notesQuery ] );
+	const latestNotes = getNotes( UNREAD_NOTES_QUERY );
```

Comment:
This is weird, why don't we just call getNotes without assigning it to a variable and the lint exception can be removed? Again I know not your code, up to you if you fix.

Reply:
Did basically as you suggested, calling `getNotes` stand alone first, and then also later when needed for the `getUnreadNotesCount`.